nx-dev/nx-dev/public/documentation/shared/workspace/buildable-and-publishable-libraries.md

```diff
@@ -31,4 +31,6 @@ Buildable libraries are similar to "publishable libraries" described above. Thei
 
 Buildable libraries are mostly used for producing some pre-compiled output that can be directly referenced from an Nx workspace application without the need to again compile it. A typical scenario is to leverage Nx’s [incremental building](/ci/incremental-builds) capabilities.
 
+Note: In order for a buildable library to be pre-compiled, it can only depend on other buildable libraries. This allows you to take full advantage of incremental builds.
```

Comment:
nitpick: This will give it better callout similar to the one here:
https://nx-dev-git-docs-buildable-dependencies-nrwl.vercel.app/guides/nx-devkit-angular-devkit#nx-devkit-and-angular-devkit

```suggestion
> Note: In order for a buildable library to be pre-compiled, it can only depend on other buildable libraries. This allows you to take full advantage of incremental builds.
```